### DISCUSSION TITLE
Laser lasert nicht komplett

### DISCUSSION THREAD

**Discussion by Fabster1000** [arkypita/LaserGRBL#898] [Q&A]
Servus Leute,

ich hab das teil neu und versteh was nicht.

Warum lasert er nicht 1:1

Siehe anhang
![1](https://user-images.githubusercontent.com/61733034/75761520-83901b80-5d39-11ea-83a6-a5e4616c8598.JPG)
![img_3245](https://user-images.githubusercontent.com/61733034/75761524-8428b200-5d39-11ea-82a6-432ed5f694f8.jpg)

Danke mal für eure Tips

**Comment by arkypita** [comment]
Size not match?

https://github.com/arkypita/LaserGRBL/issues/628
